index.bs

```diff
@@ -0,0 +1,158 @@
+<pre class='metadata'>
+Title: Runtime Keys
+Shortname: runtime-keys
+Level: 1
+Status: w3c/CG-DRAFT
+Group: wintercg
+URL: http://runtime-keys.proposal.wintercg.org
+Editor: Ethan Arrowood, Vercel http://vercel.com, ethan-arrowood@vercel.com
+Abstract: A proposal defining standard identifier keys for various runtime environments
+Markup Shorthands: markdown yes
+</pre>
+<pre class=link-defaults>
+spec:url; type:interface; text:URL
+spec:html; type:attribute; for:Window; text:navigator
+spec:html; type:attribute; for:Window; text:self
+</pre>
+
+Introduction {#intro}
+=====================
+
+This proposal defines a list of keys that represent various runtime environments. These keys can be used in a variety of ways, and should be considered a reliable representation of the given runtime. For example, they may be used in a project configuration file to indicate the given project supports the specified runtime. This specification does not detail <i>how</i> the keys can be used; that is left up to community developed tools. The point of this proposal is too only define <i>what</i> the keys are in order to prevent conflicts and give users a reliable list of platforms they can build tooling around.
+
+Keys {#keys}
+============
+
+## Cloudflare - workerd ## {#workerd}
+
+The JavaScript / Wasm runtime that powers Cloudflare Workers.
+
+Key: `workerd`
+
+Website: [https://workers.cloudflare.com/](https://workers.cloudflare.com/)
+
+Repository: [https://github.com/cloudflare/workerd](https://github.com/cloudflare/workerd)
+
+## Deno Land - Deno ## {#deno}
+
+A modern runtime for JavaScript and TypeScript.
+
+Key: `deno`
+
+Website: [https://deno.com](https://deno.com)
+
+Repository: [https://github.com/denoland/deno](https://github.com/denoland/deno)
+
+## Meta - React Native ## {#react-native}
+
+Create native apps for Android and iOS using React.
+
+Key: `react-native`
+
+Website: [https://reactnative.dev/](https://reactnative.dev/)
+
+Repository: [https://github.com/facebook/react-native](https://github.com/facebook/react-native)
+
+## Netlify - Edge Functions ## {#}
```

Comment:
```suggestion
## Netlify - Edge Functions ## {#netlify}
```
should we at least have a temporary naming for this one ? and same for key. I think we choose for others runtimes without having someone from the company so why not here too ?

Reply:
Other ones were pulled from existing resources or told to me by someone from that project. I've reached out to someone from Netlify so they will be checking out soon